network/discovery/discovery.go

```diff
@@ -299,7 +298,7 @@ func (d *DiscoveryService) findPeersCall(
 
 	clt, clientErr := d.baseServer.NewDiscoveryClient(peerID)
 	if clientErr != nil {
-		return nil, fmt.Errorf("unable to create new discovery client connection, %w", clientErr)
+		return nil, clientErr
```

Comment:
Why stop wrapping error with the context?
The previous error is more readable.

Reply:
[discovery.go#L383](https://github.com/dogechain-lab/dogechain/blob/b2cfd24ba7d31b6c008a2fa972510074337fbcde/network/discovery/discovery.go#L383) and [discovery.go#L435](https://github.com/dogechain-lab/dogechain/blob/b2cfd24ba7d31b6c008a2fa972510074337fbcde/network/discovery/discovery.go#L435), print error is too long,  [server_discovery.go#L53](https://github.com/dogechain-lab/dogechain/blob/b2cfd24ba7d31b6c008a2fa972510074337fbcde/network/server_discovery.go#L53) exist are duplicate descriptions